package.json

```diff
@@ -14,6 +14,7 @@
     "packages": [
       "apps/**/*",
       "configs/**/*",
+      "scripts/**/*",
```

Comment:
I'd be tempted to move this to `packages/tools`, as this seems like it fits in that category of packages potentially (and keen not to have too many folders at the root of the repo).

Could even use `packages/tools/scripts` if we think we're likely to have more than one github action script there in the future

Reply:
I think that also makes sense. 👍 

I don't know if it's a concern only I have but I wished we had a clearer separation in directories for consumer packages and internal ones.

Reply:
I don't think we've looked at it yet, but if we wanted to rearrange directories to reflect this in the future we could do that.

If you've got any opinions on how it could work, might be worth opening a Github discussion and can think it through there?